### PR TITLE
adding flag to install packages at runtime

### PR DESCRIPTION
**Pull request by vicki-c**
- we need `aws-cli` installed for the helm provider
- opting to do this rather than building new images for different versions of the terraform image

### CODE REVIEW

**Comment by dizzzan** [comment]
Can we update the docs to add the new flag? And maybe the tests 😄

File: plugin.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ configuration:
       type: array
     volumes:
       type: array
-    volumes:
```
Comment (dizzzan) [inline plugin.yml:65]: convenient placeholder 😆

File: hooks/command [hunk 2]
```diff
@@ -188,14 +219,16 @@ EOF
...
   local COMMAND_BLOCK=""
   local ENVIRONMENT_BLOCK=""
   local VOLUMES_BLOCK=""
 
   for cmd in "$@"; do
+    ENTRYPOINT_BLOCK=$(get_docker_entrypoint)
     COMMAND_BLOCK=$(get_docker_command "$cmd")
     ENVIRONMENT_BLOCK=$(get_environment_block "$cmd")
-    VOLUMES_BLOCK=$(get_volumes_block "$cmd")
+    VOLUMES_BLOCK=$(get_volumes_block "$ENTRYPOINT_BLOCK")
```
Comment (dizzzan) [inline hooks/command:230]: is this intended? i dont really understand whats going on here

Comment (vicki-c) [reply hooks/command:230]: it actually looks like it never used the input. so i just replaced it with the input that i want

Comment (dizzzan) [reply hooks/command:230]: doesn't that end up duplicating the block [here?](https://github.com/gantry-ml/simple-terraform-buildkite-plugin/pull/8/files#diff-94a101319341e0a2056fd83b6ec5656a8464b1af49f7192644132851e1256de6R251-R254)

Comment (vicki-c) [reply hooks/command:230]: `get_volumes_block` doesn't actually spit `ENTRYPOINT_BLOCK` back out, so no. i can see how it's confusing tho

**Review by dizzzan — APPROVED** [review]

**Pull request merged**